lib/controllers/flow/account_linking_flow_controller.dart

```diff
@@ -69,7 +70,9 @@ class AccountLinkingFlowController extends GetxController {
           // The consent data has been updated
           _setAwaitingUpdate(false);
 
-          // TODO(kkzeng): Display screen with list of accounts to link here
+          // Redirect to the next stage in account linking flow
+          // Display list of associated accounts
+          Get.to<dynamic>(AssociatedAccounts(this));
```

Comment:
Could there be a better name? This looks like a data object name instead of screen... but yeah I couldnt think for one now sorry :| 

Also, just realized about this... but I think we need to add `Screen` suffix to the name of the classes that are used to represent a screen.

Reply:
Just renamed it to `DiscoveredAccountsScreen`. Hopefully that's clearer

Reply:
I feel like it would be better to name it using something that could describe the action rather than the data.

For example, `PaymentInitiation`, `PaymentConfirmation`, and `PaymentAuthorization` all sound like a process that user will do with a screen, but `DiscoveredAccount` feels more like a list of accounts that you get from the discovery process. Well, this is just based on my intuition, but yeah up to you if you feel like its a more suitable name :-)